README.md

```diff
@@ -287,4 +288,5 @@ interface XRWebGL2GraphicsBinding {
   - Sensible defaults of all the layer values.
   - Do we need an attribte to communicate max number of supported layers?
   - Do we need to report on the layer source whether or not it's mono/stereo, since the XR device may force a downgrade?
-  - Depth: Should we consider allowing a separate layer for this like OpenXR does, or continue pattern of allowing it's use by default (implies most layer types need to be able to allocate both a color and depth/stencil buffer)
+  - Depth: Should we consider allowing a separate layer for this like OpenXR does, or continue pattern of allowing it's use by default (implies most layer types need to be able to allocate both a color and depth/stencil buffer)-
+  - Should there be a way to explicitly dispose of a layer?
```

Comment:
Layer would be eventually cleaned up by the JS VM but that might take too long. 
I'm also unsure if JS knows how much memory they take up.

Reply:
That sounds like a really good idea to me.